src/main/java/lotto/domain/LottoWin.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+public class LottoWin {
+    private static int WIN_EARNING_BY_THREE = 5000;
+    private static int WIN_EARNING_BY_FOUR = 50000;
+    private static int WIN_EARNING_BY_FIVE = 1500000;
+    private static int WIN_EARNING_BY_SIX = 2000000000;
```

Comment:
Java 7부터는 `100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
    private static int WIN_EARNING_BY_SIX = 2_000_000_000;
```